project/Dependencies.scala

```diff
@@ -34,7 +34,7 @@ object Dependencies {
   lazy val infoset = Seq(
     "org.jdom" % "jdom2" % "2.0.6",
     "com.fasterxml.woodstox" % "woodstox-core" % "5.1.0",
-    "com.fasterxml.jackson.core" % "jackson-core" % "2.9.8"
+    "com.fasterxml.jackson.core" % "jackson-core" % "2.10.2"
```

Comment:
There is one more file to update which is bin.NOTICE in daffodil-cli. 
One must also verify that the license for jackson-core hasn't changed in the new version.

Reply:
I've updated bin.NOTICE: https://github.com/apache/incubator-daffodil/pull/328/files#diff-1f1ec1d6ff78037b56d3b4855b5e75e4R63 (probably not in the first patch, I've done a force push since)
The license haven't changed for 7 years: https://github.com/FasterXML/jackson-databind/blob/jackson-databind-2.10.2/src/main/resources/META-INF/LICENSE